packages/spacecat-shared-data-access/src/service/import-url/accessPatterns.js

```diff
@@ -71,3 +71,28 @@ export const updateImportUrl = async (dynamoClient, config, log, importUrl) => {
 
   return importUrl;
 };
+
+/**
+ * Get Import Urls by Job ID and Status
+ * @param {DynamoClient} dynamoClient
+ * @param {Object} config
+ * @param {Logger} log
+ * @param {string} jobId
+ * @param {string} status
+ * @returns {Promise<ImportUrlDto[]>}
+ */
+export const getImportUrlsByJobIdAndStatus = async (dynamoClient, config, log, jobId, status) => {
+  const items = await dynamoClient.query({
+    TableName: config.tableNameImportUrls,
+    IndexName: config.indexNameImportUrlsByJobIdAndStatus,
+    KeyConditionExpression: 'jobId = :jobId AND #status = :status',
+    ExpressionAttributeNames: {
+      '#status': 'status',
+    },
```

Comment:
this is probably redundant when using the right attribute name directly in the `KeyConditionExpression`

Reply:
Yes. `status` is a reserved keyword. Had to add it under `ExpressionAttributeNames` for the query to go through.